File: Programming_Basics_with_Python/conditional_statements_advanced_exercise/07._hotel_room.py
month = input()
overnight = int(input())
studio = 0
apartment = 0

if month == "May" or month == "October":
    studio = 50
    apartment = 65
    if overnight > 14:
        studio = studio - studio * 0.30
        apartment = apartment - apartment * 0.10
    elif overnight > 7:
        studio = studio - studio * 0.05
elif month == "June" or month == "September":
    studio = 75.20
    apartment = 68.70
    if overnight > 14:
        studio = studio - studio * 0.20
        apartment = apartment - apartment * 0.10
elif month == "July" or month == "August":
    studio = 76
    apartment = 77
    if overnight > 14:
        apartment = apartment - apartment * 0.10

print(f"Apartment: {overnight * apartment:.2f} lv.")
print(f"Studio: {overnight * studio:.2f} lv.")

File: Programing_Basics_with_JS/conditionalStatementsAdvancedExercise/07.HotelRoom.js
function hotelRoom(input) {
    let month = input[0];
    let nightCount = Number(input[1]);

    let costForApartment = 0;
    let costForStudio = 0;

    switch(month) {
        case 'May':
        case 'October':
            costForApartment = nightCount * 65;
            costForStudio = nightCount * 50;
            if (nightCount > 7 && nightCount <= 14) {
                costForStudio = costForStudio * 0.95;
            } else if (nightCount > 14) {
                costForStudio = costForStudio * 0.70;
            } break;
            
        case 'June':
        case 'September':
            costForApartment = nightCount * 68.70;
            costForStudio = nightCount * 75.20;
            if (nightCount > 14) {
                costForStudio = costForStudio * 0.80;
            } break;

        case 'July':
        case 'August':
            costForApartment = nightCount * 77;
            costForStudio = nightCount * 76;
            break;
    }

    if (nightCount > 14) {
        costForApartment = costForApartment * 0.90;
    }

    console.log(`Apartment: ${costForApartment.toFixed(2)} lv.`);
    console.log(`Studio: ${costForStudio.toFixed(2)} lv.`);
}

hotelRoom(["May",
"15"])

hotelRoom(["June",
"14"])

hotelRoom(["August",
"20"])

